ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -9,12 +9,14 @@ import SwiftUI
 /// Properties that make up cell content
 @objc public class MSFListCellState: NSObject, ObservableObject, Identifiable {
     public var id = UUID()
-    @objc @Published public var leadingView: UIView?
+    @objc @Published public var leadingView: UIView? // Supporting square views only
```

Comment:
Same thing about the documentation comment as below. The only thing I'd change in the wording is that this is following the "iconSize tokens" instead of "square". (iconSize is currently square, but it could change in the future).

As another nit comment on the naming not specifically related to this PR, how about changing the name from "iconSize" to "leadingViewSize"? I think it makes more sense after we replaced the property from an image to a view.